util/table-filter/parser.go

```diff
@@ -58,7 +58,7 @@ func (p *tableRulesParser) parse(line string, canImport bool) error {
 		return err
 	}
 	if len(line) == 0 {
-		return p.errorf("missing table pattern")
+		return p.errorf("wrong table pattern")
```

Comment:
Currently, the dataflow-service uses `missing table pattern` as key words to identify the pattern error. Maybe we should note it in case the DFS's unexpected malfunctioning.